viewers/wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/pages/bootstrap-overrides-all-v2.css

```diff
@@ -25,30 +25,34 @@
 }
 
 #body {
-    padding-top: 0px;
+    padding-top: 61px;
```

Comment:
Not sure this covers all the cases, but looks ok for now

Reply:
Without knowing any context, seems it would be cool if we don't need that.

Reply:
Yeah... the fixed-top class is the culprit right now, don't know how to address it otherwise

Reply:
I looked up some examples on bootswatch and they seem to be adding 120px top-padding in their custom CSS for the theme demos: 
https://bootswatch.com/_assets/css/custom.min.css